.github/workflows/pull-request.yml

```diff
@@ -13,7 +13,4 @@ jobs:
         uses: ballerina-platform/ballerina-library/.github/workflows/pull-request-build-template.yml@main
         secrets: inherit
         with:
-            additional-ubuntu-build-flags: "-x test"
-            additional-ubuntu-test-flags: "-x test"
-            additional-windows-build-flags: "-x test"
             additional-windows-test-flags: "-x test"
```

Comment:
Any reason to disable windows tests here?

Reply:
The docker image which I previously used to run the ldap server is not supported for Windows platform. It is fixed by adding this check. It is not needed now. Fixed in 4cd6dfe

Reply:
Skip tests flag is still here?

Reply:
Revert the fix as Windows runner does not support running docker images